sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -570,20 +631,168 @@ public ServiceBusReceiverClientBuilder subscriptionName(String subscriptionName)
+                    tracerProvider, messageSerializer, ServiceBusClientBuilder.this::onClientClose);
+            }
+        }
+
+        /**
+         * Creates a <b>synchronous</b>, <b>session-aware</b> Service Bus receiver responsible for reading {@link
+         * ServiceBusMessage messages} from a specific queue or topic.
+         *
+         * @return An new {@link ServiceBusReceiverClient} that receives messages from a queue or topic.
+         * @throws IllegalStateException if {@link #queueName(String) queueName} or {@link #topicName(String)
+         *     topicName} are not set or, both of these fields are set. It is also thrown if the Service Bus {@link
+         *     #connectionString(String) connectionString} contains an {@code EntityPath} that does not match one set in
+         *     {@link #queueName(String) queueName} or {@link #topicName(String) topicName}. Lastly, if a {@link
+         *     #topicName(String) topicName} is set, but {@link #subscriptionName(String) subscriptionName} is not.
+         * @throws IllegalArgumentException Queue or topic name are not set via {@link #queueName(String)
+         *     queueName()} or {@link #topicName(String) topicName()}, respectively.
+         */
+        public ServiceBusReceiverClient buildClient() {
+            return new ServiceBusReceiverClient(buildAsyncClient(), retryOptions.getTryTimeout());
+        }
+
+        /**
+         * This is a rolling session receiver only if maxConcurrentSessions is > 0 AND sessionId is null or empty. If
+         * there is a sessionId, this is going to be a single, named session receiver.
+         *
+         * @return {@code true} if this is an unnamed rolling session receiver; {@code false} otherwise.
+         */
+        private boolean isRollingSessionReceiver() {
+            if (maxConcurrentSessions == null) {
+                return false;
+            }
+
+            if (maxConcurrentSessions < 1) {
+                throw logger.logExceptionAsError(
+                    new IllegalArgumentException("Maximum number of concurrent sessions must be positive."));
+            }
+
+            return CoreUtils.isNullOrEmpty(sessionId);
+        }
+    }
+
+    /**
+     * Builder for creating {@link ServiceBusReceiverClient} and {@link ServiceBusReceiverAsyncClient} to consume
+     * messages from Service Bus.
+     */
+    @ServiceClientBuilder(serviceClients = {ServiceBusReceiverClient.class, ServiceBusReceiverAsyncClient.class})
+    public final class ServiceBusReceiverClientBuilder {
+        private int prefetchCount = DEFAULT_PREFETCH_COUNT;
+        private String queueName;
+        private ReceiveMode receiveMode = ReceiveMode.PEEK_LOCK;
+        private String subscriptionName;
+        private String topicName;
+
+        private ServiceBusReceiverClientBuilder() {
+        }
+
+        /**
+         * Sets the prefetch count of the receiver. Prefetch speeds up the message flow by aiming to have a message
```

Comment:
Lot of user do not read till end of java doc. Can we say in first line that ` default value is 1.` Something like this `Sets the prefetch count of the receiver otherwise defaults to 1.`